guides/source/upgrading_ruby_on_rails.md

```diff
@@ -100,6 +100,21 @@ To:
 Rails.application.secrets[:smtp_settings][:address]
 ```
 
+### Since 5.1 migrations have to be versioned
+
+In 5.0 when you generated migration it would get a version (the migration class
```

Comment:
If I could make a copy suggestion, I would rewrite this as

As of Rails 5.0, generated migrations inherit from a specific version of ActiveRecord's migration class (eg. `ActiveRecord::Migration[5.0]`). Versioning was introduced as a means to keep migration SQL consistent across Rails upgrades. Migrations generated using Rails 4 and below will need to be updated to inherit from a versioned migration class (eg. `ActiveRecord::Migration[4.2]`) in order to runnable on Rails 5.1.